tests/trace/test_client_trace.py

```diff
@@ -1561,23 +1562,6 @@ class MySerializableClass(weave.Object):
     assert b2.obj == repr(b_obj)
 
 
-# Note: this test only works with the `trace_init_client` fixture
-@pytest.mark.skip(reason="TODO: Skipping since it seems to rely on the testcontainer")
-def test_ref_get_no_client(trace_init_client):
-    trace_client = trace_init_client.client
-    data = weave.publish(42)
-    data_got = weave.ref(data.uri()).get()
-    assert data_got == 42
-
-    # clear the graph client effectively "de-initializing it"
-    with _no_graph_client():
-        # This patching is required just to make the test path work
-        with _patched_default_initializer(trace_client):
-            # Now we will try to get the data again
-            data_got = weave.ref(data.uri()).get()
-            assert data_got == 42
-
-
```

Comment:
is this still needed?

Reply:
how long has this been skipped?

Reply:
This has only been skipped since the refactor, but it uses this `trace_init_client` that I don't see in other tests

Reply:
Maybe it can be rewritten with the normal `client` fixture